src/backend/distributed/citus.sql

```diff
@@ -194,6 +194,13 @@ COMMENT ON FUNCTION master_create_distributed_table(table_name regclass,
                                                     distribution_method citus.distribution_type)
     IS 'define the table distribution functions';
 
+CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
```

Comment:
On 2016-04-28 14:07:49 -0700, Metin Döşlü wrote:
> > This change probably requires an upgrade script.
> CREATE FUNCTION master_update_shard_statistics(relation_id regclass, shard_id bigint)
> 
> Should it be enough to move this to citus--5.0--5.0-1.sql?

No, by the time of mere that version might already have gone out to a
customer So it should be citus--5.0-1--5.0-2.sql?

> I also suspect that we will rename it to citus--5.0--5.1.sql before
> the code-freeze?

No. Once we actuall freeze we'll add one more migration to the final
version. There's customers that have installed pre-release versions,
they'd have to dump/restore in that case.